<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dataExport</class>
 <widget class="QMainWindow" name="dataExport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1248</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>dataExport</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qrc.qrc">
    <normaloff>:/src/imp.jpg</normaloff>:/src/imp.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(89, 156, 200);
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,3">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="lineEdit_refer">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit
{
border:2px solid white;
background-color: rgb(18, 204, 209);
border-radius:5px;
font:75 11pt &quot;仿宋&quot;;
color:rgb(52,101,64);
}
QLineEdit:hover
{
background-color: rgb(238, 238, 236);
}
</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="referBtn">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(85, 255, 127);
</string>
            </property>
            <property name="text">
             <string>搜索</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="tableView_tables">
          <property name="styleSheet">
           <string notr="true">QTableView
{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:1 rgba(5, 126, 158, 255));
selection-background-color:rgb(30, 167, 160);
selection-color: rgb(252, 233, 79);
font:75 11pt &quot;仿宋&quot;;
}

QTableView:hover
{
border:2px inset red;
}

QTableView::item:hover
{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(38, 218, 210, 255), stop:1 rgba(255, 255, 255, 255));
}
</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4" stretch="3,1">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>31</height>
               </size>
              </property>
              <property name="text">
               <string>数据检索：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>31</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QLineEdit
{
border:2px solid white;
background-color: rgb(18, 204, 209);
border-radius:5px;
font:75 11pt &quot;仿宋&quot;;
color:rgb(52,101,64);
}
QLineEdit:hover
{
background-color: rgb(238, 238, 236);
}
</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="searchBtn">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>31</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(85, 255, 127);
border_radius:5px;


</string>
              </property>
              <property name="text">
               <string>检索</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTableView" name="tableView">
            <property name="font">
             <font>
              <family>Microsoft YaHei UI</family>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QTableView
{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:1 rgba(81, 181, 209, 255));
alternate-background-color: rgb(135, 206, 250);
selection-background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0.476923 rgba(133, 211, 203, 255),  stop:1 rgba(255,255,255,255));
selection-color: rgb(239, 41, 41);
color:rgb(0,0,0);
}

QTableView:hover
{
border:2px inset red;
}
QTableView QTableCornerButton::section
{
background-color: rgb(173, 255, 47);
border:0px outset red;
}
QTableView::item:hover
{
background-color: rgb(0,255,0);
border:1px solid #bfcde4;
}
QTableView::item:selected
{
border:1px outset #567dbc;
}
</string>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <attribute name="verticalHeaderCascadingSectionResizes">
             <bool>false</bool>
            </attribute>
            <attribute name="verticalHeaderDefaultSectionSize">
             <number>25</number>
            </attribute>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QPushButton" name="addRecBtn">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>31</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>75</width>
                <height>30</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(0, 255, 255);
</string>
              </property>
              <property name="text">
               <string>添加记录</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>31</height>
               </size>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>107</width>
                <height>28</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="deletRecBtn">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>31</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>75</width>
                <height>30</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(0, 255, 255);</string>
              </property>
              <property name="text">
               <string>删除记录</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>31</height>
               </size>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>28</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="QPushButton" name="pushButton">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>31</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>25</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>||&lt;</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_2">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>31</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>25</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>&lt;&lt;</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_2">
                <property name="minimumSize">
                 <size>
                  <width>40</width>
                  <height>31</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>40</width>
                  <height>31</height>
                 </size>
                </property>
                <property name="text">
                 <string>当前第</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_6">
                <property name="minimumSize">
                 <size>
                  <width>35</width>
                  <height>31</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>35</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">QLineEdit
{
border:2px solid white;
background-color: rgb(18, 204, 209);
border-radius:5px;
font:75 11pt &quot;仿宋&quot;;
color:rgb(52,101,64);
}
QLineEdit:hover
{
background-color: rgb(238, 238, 236);
}
</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_3">
                <property name="minimumSize">
                 <size>
                  <width>70</width>
                  <height>31</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>70</width>
                  <height>31</height>
                 </size>
                </property>
                <property name="text">
                 <string>页，共1页。</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_3">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>31</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>25</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>&gt;&gt;</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_4">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>31</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>25</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>&gt;||</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>31</height>
               </size>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>28</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="subRecBtn">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>31</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>75</width>
                <height>30</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(0, 255, 255);</string>
              </property>
              <property name="text">
               <string>提交</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>31</height>
               </size>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>28</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="commitRecBtn">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>31</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>75</width>
                <height>30</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(0, 255, 255);</string>
              </property>
              <property name="text">
               <string>保存</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,4,2,1">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_9">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>31</height>
               </size>
              </property>
              <property name="text">
               <string>导出路径：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_5">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>31</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QLineEdit
{
border:2px solid white;
background-color: rgb(18, 204, 209);
border-radius:5px;
font:75 11pt &quot;仿宋&quot;;
color:rgb(52,101,64);
}
QLineEdit:hover
{
background-color:green;
color:red;
}
</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox">
              <property name="minimumSize">
               <size>
                <width>55</width>
                <height>31</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>55</width>
                <height>31</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>格式</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>*.sql</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>*.xml</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="exportBtn_4">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>31</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(85, 255, 127);
</string>
              </property>
              <property name="text">
               <string>导出</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QListView" name="listView">
            <property name="styleSheet">
             <string notr="true">QListView
{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:1 rgba(18, 83, 103, 255));
selection-background-color: rgb(12, 232, 17);
selection-color: rgb(239, 41, 41);
color:rgb(63,230,235);
font:75 11pt &quot;仿宋&quot;;
border-top:4px solid rgb(63,230,235);
}

QListView::item
{
min-height:20px;
}
</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="textEdit">
            <property name="styleSheet">
             <string notr="true">
QTextEdit
{
selection-background-color: rgb(0, 255, 255);
selection-color: rgb(255, 0, 0);
border:2px solid green;
border-radius:2px;
}
</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>导出进度：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QProgressBar" name="progressBar_4">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QProgressBar{
background-color: rgb(127, 255, 0);
}

QProgressBar::chunk{
background-color:green;
width:10px;
margin:0.5px;
}</string>
              </property>
              <property name="value">
               <number>20</number>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1248</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QMenuBar
{
background-color: qlineargradient(spread:pad, x1:0, y1:0.0222727, x2:1, y2:1, stop:0.306818 rgba(37, 159, 0, 255), stop:0.943182 rgba(35, 223, 223, 255));
}

QMenuBar::item
{
spacing:3px;
padding:2px 4px;
border-radius:4px;
background-color:transparent;
}

QMenuBar::item:selected
{
color:#008080;
background-color:#e0ffff;
}

</string>
   </property>
   <widget class="QMenu" name="menu">
    <property name="styleSheet">
     <string notr="true">QMenu
{
background-color:#b0e0e6;
margin:2px;
}

QMenu::item
{
padding:2px 25px 2px 20px;
border:2px solid #9acd32;
}

QMenu::item:selected
{
color:#ff6347;
background-color: #00cede;
border:1px solid rgba(82,130,164,1);
}

</string>
    </property>
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actiondakai"/>
    <addaction name="actionlingcunwei"/>
    <addaction name="actionexit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="styleSheet">
     <string notr="true">QMenu
{
background-color:#b0e0e6;
margin:2px;
}

QMenu::item
{
padding:2px 25px 2px 20px;
border:2px solid #9acd32;
}

QMenu::item:selected
{
color:#ff6347;
background-color: #00cede;
border:1px solid rgba(82,130,164,1);
}

</string>
    </property>
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="actionfuzhi"/>
    <addaction name="actionzhantie"/>
    <addaction name="actionjiantie"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="styleSheet">
     <string notr="true">QMenu
{
background-color:#b0e0e6;
margin:2px;
}

QMenu::item
{
padding:2px 25px 2px 20px;
border:2px solid #9acd32;
}

QMenu::item:selected
{
color:#ff6347;
background-color: #00cede;
border:1px solid rgba(82,130,164,1);
}

</string>
    </property>
    <property name="title">
     <string>帮助</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="styleSheet">
    <string notr="true">QToolBar
{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:1 rgba(81, 170, 204, 255));
spacing:2px;
}
QToolButton
{
border:5px solid #008b8b;
border-radius:6px;
color:red;
}
QToolButton[popupMode=&quot;1&quot;]
{
padding-right:20px;
}

QToolButton:pressed
{
background-color: #20b2aa;
}

QToolButton:hover
{
border:5px solid #00ced1;
color:green;
}
</string>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_condb"/>
   <addaction name="action_dataimp"/>
   <addaction name="action_sfexit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_condb">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/src/connectdb.jpg</normaloff>:/src/connectdb.jpg</iconset>
   </property>
   <property name="text">
    <string>连接数据库</string>
   </property>
   <property name="toolTip">
    <string>连接数据库</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetWithChildrenShortcut</enum>
   </property>
   <property name="menuRole">
    <enum>QAction::ApplicationSpecificRole</enum>
   </property>
   <property name="priority">
    <enum>QAction::LowPriority</enum>
   </property>
  </action>
  <action name="action_dataimp">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/src/exp.jpg</normaloff>:/src/exp.jpg</iconset>
   </property>
   <property name="text">
    <string>数据导入</string>
   </property>
   <property name="toolTip">
    <string>进入数据导入</string>
   </property>
   <property name="font">
    <font/>
   </property>
  </action>
  <action name="action_sfexit">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/src/exit.jpg</normaloff>:/src/exit.jpg</iconset>
   </property>
   <property name="text">
    <string>退出软件</string>
   </property>
   <property name="toolTip">
    <string>退出软件</string>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actiondakai">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actionlingcunwei">
   <property name="text">
    <string>另存为</string>
   </property>
  </action>
  <action name="actionexit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionfuzhi">
   <property name="text">
    <string>复制</string>
   </property>
  </action>
  <action name="actionzhantie">
   <property name="text">
    <string>粘贴</string>
   </property>
  </action>
  <action name="actionjiantie">
   <property name="text">
    <string>剪贴</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qrc.qrc"/>
 </resources>
 <connections/>
</ui>
